documentapi/src/main/java/com/yahoo/documentapi/messagebus/protocol/SlobrokPolicy.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
```

Comment:
Consider updating copyright since this is a new file

Reply:
Fixed